tools/doc/json.js

```diff
@@ -215,7 +215,8 @@ function processList(section) {
   // shove the name in there for properties, since they are always
   // just going to be the value etc.
   if (section.type === 'property' && values[0]) {
-    values[0].textRaw = '`' + section.name + '` ' + values[0].textRaw;
+    //values[0].textRaw = '`' + section.name + '` ' + values[0].textRaw;
```

Comment:
This is the trickiest part, the JSON parser is definitely old and not documented. I think we can add a property without editing `textRaw`, but I have to try it yet. I'll rather not modify or explain this change :smile: